docs/file_formats.md

```diff
@@ -0,0 +1,64 @@
+File Formats
+======
+These notes cover imaging data you'll load into Caiman for analysis, typically 2d (occasionally 3d) data over time. There are several formats we support, some with caveats. If there are other formats you want, or caveats you find particularly burdensome, reach out and if it's easy we may be able to adjust the software, or alternatively offer ideas on how to convert your files for use with Caiman.
+
+
+If you are comfortable programming, you will find the code to load the movies in caiman/base/movies.py, spread over a few functions:
+
+- load()
+- load\_iter()
+- get\_file\_size()
+
+If you want to implement your own handlers with a fork of our codebase, or submit a PR to us, start there. Caiman converts files from these formats into its own internal formats during motion correction.
+
+Notes for specific formats follow:
+### Matlab (\*.mat)
+Caiman may be able to handle Matlab's mat files, although it is not a desirable format. We rely on the scipy.io.matlab functions to handle files of this type, except for certain versions of the format which are actually hdf5 files
+
+### TIFF (\*.tiff, \*.tif, \*.btf)
+Caiman can handle many TIFF files, but TIFF is a very flexible file format and we do not handle every variant. Because calcium imaging data has a time element, there are many ways that files might saved in this format (series, pages, some other things). We don't anticipate every variant.
```

Comment:
Perhaps say the best thing to do is just try, and if it doesn't work then use @EricThomson tiffit to convert it? https://github.com/EricThomson/tiffit